boilerplate/flyte/end2end/run-tests.py

```diff
@@ -258,6 +260,18 @@ def run(
     is_flag=True,
     help="Abort failing workflows upon exit",
 )
+@click.option(
+    "--test_project_name",
+    default="flytesnacks",
+    is_flag=False,
```

Comment:
is this not the default?

Reply:
`flytesnacks` is the default yeah

Reply:
oh sorry, I mean is `is_flag=False` already the default?
but no problem being explicit!